helix-core/src/main/java/org/apache/helix/HelixManagerProperty.java

```diff
@@ -54,12 +56,13 @@ public HelixManagerProperty(Properties helixManagerProperties, CloudConfig cloud
         helixManagerProperties.getProperty(SystemPropertyKeys.PARTICIPANT_HEALTH_REPORT_LATENCY));
   }
 
-  private HelixManagerProperty(String version, long healthReportLatency,
+  private HelixManagerProperty(String version, long healthReportLatency, int msgHandlerResetTimeout,
```

Comment:
Same as above: I have concerns over adding this timeout in HelixManagerProperty, due to the connectionConfig it carries.